src/main/java/org/mockbukkit/mockbukkit/entity/LivingEntityMock.java

```diff
@@ -822,11 +785,7 @@ public boolean teleport(@NotNull Location location, PlayerTeleportEvent.@NotNull
 	@Override
 	public boolean isLeashed()
 	{
-		if (!(this.leashHolder instanceof Mob))
-		{
-			return false;
-		}
-		return this.leashHolder != null;
+		return this.leashHolder instanceof Mob;
```

Comment:
I think this is another no-op change. 

This is the code in paper:
```
return this.getHandle().getLeashHolder() != null;
```
See PaperLeashable

Both implementations are wrong, but that fix is really fit for a different PR

Reply:
Nah, it's fine. I will make an issue on this one though that probably wont be patched by me at least